osu.Game/Overlays/Mods/ModCustomisationHeader.cs

```diff
@@ -0,0 +1,97 @@
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osu.Game.Localisation;
+
+namespace osu.Game.Overlays.Mods
+{
+    public partial class ModCustomisationHeader : OsuHoverContainer
+    {
+        public override bool HandlePositionalInput => true;
+
+        private Box background = null!;
+        private SpriteIcon icon = null!;
+
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; } = null!;
+
+        protected override IEnumerable<Drawable> EffectTargets => new[] { background };
+
+        public readonly BindableBool Expanded = new BindableBool();
+
+        public ModCustomisationHeader()
+        {
+            Action = Expanded.Toggle;
+            Enabled.Value = false;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            CornerRadius = 10f;
+            Masking = true;
+
+            Children = new Drawable[]
+            {
+                background = new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                },
+                new OsuSpriteText
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Text = ModSelectOverlayStrings.CustomisationPanelHeader,
+                    UseFullGlyphHeight = false,
+                    Font = OsuFont.Torus.With(size: 20f, weight: FontWeight.SemiBold),
+                    Margin = new MarginPadding { Left = 20f },
+                },
+                new Container
+                {
+                    Anchor = Anchor.CentreRight,
+                    Origin = Anchor.CentreRight,
+                    Size = new Vector2(16f),
+                    Margin = new MarginPadding { Right = 20f },
+                    Child = icon = new SpriteIcon
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Icon = FontAwesome.Solid.ChevronDown,
+                        RelativeSizeAxes = Axes.Both,
+                    }
+                }
+            };
+
+            IdleColour = colourProvider.Dark3;
+            HoverColour = colourProvider.Light4;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Enabled.BindValueChanged(e =>
+            {
+                TooltipText = e.NewValue
+                    ? string.Empty
+                    : ModSelectOverlayStrings.CustomisationPanelDisabledReason;
+            }, true);
+
+            Expanded.BindValueChanged(v =>
+            {
+                icon.RotateTo(v.NewValue ? 180 : 0);
```

Comment:
Maybe port this to regular dropdowns for consistent UI behavior? In a separate PR of course.

Reply:
If we're keeping it, I'd propose this instead:

```diff
diff --git a/osu.Game/Overlays/Mods/ModCustomisationHeader.cs b/osu.Game/Overlays/Mods/ModCustomisationHeader.cs
index 2887b53548..c96f9da00f 100644
--- a/osu.Game/Overlays/Mods/ModCustomisationHeader.cs
+++ b/osu.Game/Overlays/Mods/ModCustomisationHeader.cs
@@ -90,7 +90,7 @@ protected override void LoadComplete()
 
             Expanded.BindValueChanged(v =>
             {
-                icon.RotateTo(v.NewValue ? 180 : 0);
+                icon.ScaleTo(v.NewValue ? new Vector2(1, -1) : Vector2.One, 300, Easing.OutQuint);
             }, true);
         }
     }

```

Reply:
Applied in https://github.com/ppy/osu/pull/28553/commits/22f2f8369521abf943db38387ed173d884f6b0ea and opened separate PR to update the other UI components to follow suit (https://github.com/ppy/osu/pull/28762).